aiidalab_widgets_base/computational_resources.py

```diff
@@ -1059,23 +1058,24 @@ def on_setup_code(self, _=None):
                 "label",
                 "computer",
                 "description",
-                "input_plugin",
-                "remote_abs_path",
+                "default_calc_job_plugin",
+                "filepath_executable",
                 "use_double_quotes",
                 "prepend_text",
                 "append_text",
             ]
 
             kwargs = {key: getattr(self, key).value for key in items_to_configure}
-            kwargs["code_type"] = CodeBuilder.CodeType.ON_COMPUTER
 
             # Checking if the code with this name already exists
             qb = orm.QueryBuilder()
             qb.append(
                 orm.Computer, filters={"uuid": kwargs["computer"].uuid}, tag="computer"
             )
             qb.append(
-                orm.Code, with_computer="computer", filters={"label": kwargs["label"]}
+                orm.AbstractCode,
```

Comment:
Should this be `InstalledCode`?  InstalledCode is used on line 1084, so currently the code seems a bit inconsistent.